health/notifications/email/README.md

```diff
@@ -30,4 +30,26 @@ sudo su -s /bin/bash netdata
 
 Where `[ROLE]` is the role you want to test. The default (if you don't give a `[ROLE]`) is `sysadmin`.
 
+# Simple SMTP transport configuration
+
+If you want an alternative of sendmail in order to have a simple MTA configuration for sending emails and auth to an existing SMTP server :
+
+- Remove all sendmail stuff.
+- Install msmtp.
+- Make a symbolic link of /usr/bin/msmtp to /usr/sbin/sendmail (default debian location of sendmail) :
+```sh
+(sudo) ln -s /usr/bin/msmtp /usr/sbin/sendmail
+```
+- Login as netdata :
+```sh
+(sudo) su -s /bin/bash netdata
+```
+- Create and make configuration of ~/.msmtprc (look at the doc it's simple : https://marlam.de/msmtp/documentation/).
```

Comment:
Create the configuration of `~/.msmtprc` as shown [here](https://marlam.de/msmtp/documentation/).

Reply:
yup